### ISSUE TITLE
Fastmail "show details" broken

### ISSUE THREAD

**Issue by eam** [mozilla/contain-facebook#886]
- Facebook Container Version: 2.3.5
- Operating System + Version: Ubuntu 22.04.1 LTS
- Firefox Version: Firefox 104.0.2 (64 bit) Mozilla Firefox Snap for Ubuntu canonical-002-1.0
- Other installed Add-ons + Version + Enabled/Disabled-Status: Tested with all other extensions disabled. (I'd rather not share the full list if possible)

### Actual behavior

Clicking on the "show details" link in the Fastmail webui does not show details (expanding From/To/Cc/Subject/etc beyond the first line).

### Expected behavior

Clicking this link should expand the details in the fastmail webui

### Steps to reproduce
1. Enable Facebook Container
2. Open a new tab and read an email on Fastmail
3. Click the "show details" link

### Notes

I have been using Facebook Container on this platform for a long time without issue. I am certain a recent update caused this issue - within the last month or two.

**Comment by joshbode** [comment]
I have also have this exact issue, which I've seen sporadically in the past (I even raised an issue with Fastmail support, but closed it when the issue went away). Note - this issue occurs in the Fastmail beta site (betaapp.fastmail.com) also.

It took ages to diagnose because even though I suspected an add-on (e.g. uBlock, Privacy Badger, etc)
However, I never disabled this add-on, because... _why_ would it have an effect on fastmail.com? :)

**Comment by joshbode** [comment]
It _may_ be related to this CSS rule:
https://github.com/mozilla/contain-facebook/blob/9ab4eb7c2f8ad535c0f171400903140083ab156f/src/content_script.css#L352

which matches the class `is-hidden` on the `div` for the sidebar:
```
<div id="v183" class="v-Split v-Split--left is-hidden" style="position: absolute; z-index: 1; transform: translate3d(-220px, 0px, 0px); inset: 0px auto auto 0px; width: 200px; height: 100%; will-change: auto;">
```

**Comment by joshbode** [comment]
yep - the `display: none` style is definitely coming from `content_script.css`:

![image](https://user-images.githubusercontent.com/878107/200106135-1e4cfc7f-6eca-464e-b6f7-2c6f1c4145af.png)

**Comment by neilj** [comment]
(I'm a developer at Fastmail.) It looks like this extension is also causing the message list to be scrolled to the top whenever you open an email. Why is this extension modifying anything on our site at all? Shouldn't it only be applying to Facebook's domains?

**Comment by joshbode** [comment]
@neilj #896 appears to fix this behaviour, too (I've just tested this in Firefox Developer Edition)

I suspect this extension applies to all pages indiscriminately because in theory, Facebook tracking can be embedded in any page (but of course, never Fastmail!) and I'm not sure there's a way to apply (or not apply) CSS to specific pages

**Comment by maxxcrawford** [comment]
@joshbode @neilj We reviewed the PR and merged it in. It will be a part of our next release! 

Shout out @joshbode for the PR. 

_(Note - No exact word on immediate timing for that release, but I can estimate in the next few weeks)_

**Comment by joshbode** [comment]
thanks @maxxcrawford :)

**Comment by dcelasun** [comment]
@maxxcrawford can you share a build here in the meantime? With signed extensions, it's no longer trivial to make local changes and the only alternative is to disable the extension. Neither sound great if the update is weeks away.

**Comment by krystian3w** [comment]
https://github.com/mozilla/contain-facebook/issues/898

**Comment by krystian3w** [comment]
Maybe as fallback works:

```css
.is-hidden:not([id^="fbc-"]) { display: revert !important; }
```

but to works need Stylus or uBlock Origin (not "Lite => MV3" if you can not locally edit rules).

**Comment by krystian3w** [comment]
5 months later: https://github.com/mozilla/contain-facebook/issues/898#issuecomment-1507446045

**Comment by joshbode** [comment]
Hi @maxxcrawford - is there anything we can do to help prepare for a release (testing a pre-release, for example)?

**Comment by joshbode** [comment]
Also, we can probably close this particular issue, because Fastmail has already worked around the issue.

**Comment by krystian3w** [comment]
https://github.com/mozilla/contain-facebook/issues/921

**Comment by krystian3w** [comment]
#911

**Comment by krystian3w** [comment]
https://github.com/mozilla/contain-facebook/issues/919

**Comment by krystian3w** [comment]
I see someone pushed two new releases with fix `.is-hidden` issue.
